frontend/src/components/ProductAreaDropdown.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+
+// TO-DO: Add more product area options to this array
```

Comment:
an internal search for "list of PAs" should give the full list

Reply:
Added! Thanks for the resource!